src/pydicom/pixels/decoders/native.py

```diff
@@ -122,7 +127,12 @@ def _rle_decode_frame(
         green then all blue, with the bytes for each pixel ordered from
         MSB to LSB when reading left to right).
     """
-    if nr_bits % 8:
+    if nr_bits == 1 and nr_samples != 1:
+        raise NotImplementedError(
+            "Unable to decode RLE encoded pixel data with "
+            "*Bits Allocated* = 1 and *Samples Per Pixel* > 1."
```

Comment:
Sorry, nitpick `'Bits Allocated'` and `'Samples per Pixel'`

Reply:
Addressed in [866c50c](https://github.com/pydicom/pydicom/pull/2216/commits/866c50c2a6c9f526c132a49ccd72012c3ea62b33)